### PR TITLE
Fix ipfs gateway domain.

### PR DESCRIPTION
**Pull request by AyumuKasuga**
Hello my friends,
Domain `ipfs.io` does not work anymore, let's replace it to by another gateway --`dweb.link` which is also managed by Protocol Labs.

### CODE REVIEW

File: docs/build/build-tools-index.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Please see the [Wallets](../general/wallets-and-extensions.md) page.
 
 - [Polkadot-JS Apps Explorer](https://polkadot.js.org/apps/#/explorer) - Polkadot dashboard block
   explorer. Supports dozens of other networks, including Kusama, Westend, and other remote or local
-  endpoints. [Access via IPFS](https://ipfs.io/ipns/dotapps.io)
+  endpoints. [Access via IPFS](https://dweb.link/ipns/dotapps.io)
```
Comment (filippoweb3) [inline docs/build/build-tools-index.md]:
```suggestion
  endpoints. [Access via IPFS](https://cloudflare-ipfs.com/ipns/dotapps.io/)
```

File: docs/general/faq.md [hunk 2]
```diff
@@ -85,7 +85,7 @@ size of the active set and how many validators are waiting in the pool.
 There are a few ways to estimate the minimum stake.
 
 One way can be to navigate to the
-[Polkadot Apps Targets tab](https://ipfs.io/ipns/polkadot.dotapps.io/#/staking/targets). The value
+[Polkadot Apps Targets tab](https://dweb.link/ipns/polkadot.dotapps.io/#/staking/targets). The value
```
Comment (filippoweb3) [inline docs/general/faq.md]:
```suggestion
[Polkadot Apps Targets tab](https://cloudflare-ipfs.com/ipns/dotapps.io/#/staking/targets). The value
```

File: docs/maintain/kusama/maintain-guides-society-kusama.md [hunk 3]
```diff
@@ -59,7 +59,7 @@ any questions if anything is unclear.
 ### 1. Bid Phase
 
 To submit a bid, click the Submit Bid button on the
-[Society page](https://ipfs.io/ipns/kusama.dotapps.io/#/society).
+[Society page](https://dweb.link/ipns/kusama.dotapps.io/#/society).
```
Comment (filippoweb3) [inline docs/maintain/kusama/maintain-guides-society-kusama.md]:
```suggestion
[Society page](https://cloudflare-ipfs.com/ipns/dotapps.io/#/society).
```

File: docs/maintain/kusama/maintain-guides-society-kusama.md [hunk 4]
```diff
@@ -73,7 +73,7 @@ declare the bid amount (1 KSM in this case) that they will receive for joining t
 ![test_bid](../../assets/society/test_bid.jpg)
 
 Once you have submitted the transaction, your bid will be shown on the
-[Society page](https://ipfs.io/ipns/kusama.dotapps.io/#/society) under the bids section. You can
+[Society page](https://dweb.link/ipns/kusama.dotapps.io/#/society) under the bids section. You can
```
Comment (filippoweb3) [inline docs/maintain/kusama/maintain-guides-society-kusama.md]:
```suggestion
[Society page](https://cloudflare-ipfs.com/ipns/dotapps.io/#/society) under the bids section. You can
```

**Review by filippoweb3 — CHANGES_REQUESTED** [review]
Thank you! I added the official link -> https://cloudflare-ipfs.com/ipns/dotapps.io

File: docs/build/build-tools-index.md (hunk 1 above)
Comment (AyumuKasuga) [reply docs/build/build-tools-index.md]: Changed

File: docs/maintain/kusama/maintain-guides-society-kusama.md (hunk 3 above)
Comment (AyumuKasuga) [reply docs/maintain/kusama/maintain-guides-society-kusama.md]: Changed

File: docs/general/faq.md (hunk 2 above)
Comment (AyumuKasuga) [reply docs/general/faq.md]: Changed

File: docs/maintain/kusama/maintain-guides-society-kusama.md (hunk 4 above)
Comment (AyumuKasuga) [reply docs/maintain/kusama/maintain-guides-society-kusama.md]: Changed

**Review by filippoweb3 — APPROVED** [review]

**Pull request merged**